features/situation-feedback/api/src/main/java/org/opennms/features/situationfeedback/api/AlarmFeedback.java

```diff
@@ -61,8 +65,12 @@ public static FeedbackType valueOfOrUnknown(String type) {
 
     private final FeedbackType feedbackType;
 
+    private final Boolean rootCause;
```

Comment:
Only the builder should allow the root cause field to be nullable.

```suggestion
    private final boolean rootCause;
```

Reply:
it needs to be Boolean in case the json that is deserialized does not have the element.

Reply:
Thats true of the builder, but in the constructor for this object it is always explicitly set to true or false.